widget/embedded/src/components/Slippage/Slippage.tsx

```diff
@@ -69,6 +69,12 @@ export function Slippage() {
           variant="contained"
           value={customSlippage || ''}
           color="dark"
+          onKeyDown={(event) => {
+            const disallowedKeys = ['-', '+'];
+            if (disallowedKeys.includes(event.key)) {
+              event.preventDefault();
+            }
+          }}
```

Comment:
It's not within your assigned task. We're encountering a similar problem with the token amount input on the homepage. Could you address that as well?